model/tv-prescription.js

```diff
@@ -0,0 +1,17 @@
+const getTVPrescriptionModel = (sequelize, {DataTypes}) => {
+  const TVPrescription = sequelize.define('tvPrescription', {
+    message: {
+      type: DataTypes.STRING,
+      unique: false,
```

Comment:
Never used sequelize - is it standard to use `false` here instead of `DataTypes.BOOLEAN` ?